css/index.css

```diff
@@ -1,3 +1,791 @@
+
+aside,
+
+canvas,
+
+details,
+
+embed,
+
+figure,
+
+figcaption,
+
+footer,
+
+header,
+
+hgroup,
+
+menu,
+
+nav,
+
+output,
+
+ruby,
+
+section,
+
+summary,
+
+time,
+
+mark,
+
+audio,
+
+video {
+
+  margin: 0;
+
+  padding: 0;
+
+  border: 0;
+
+  font-size: 100%;
+
+  font: inherit;
+
+  vertical-align: baseline;
+
+}
+
+/* HTML5 display-role reset for older browsers */
+
+article,
+
+aside,
+
+details,
+
+figcaption,
+
+figure,
+
+footer,
+
+header,
+
+hgroup,
+
+menu,
+
+nav,
+
+section {
+
+  display: block;
+
+}
+
+body {
+
+  line-height: 1;
+
+}
+
+ol,
+
+ul {
+
+  list-style: none;
+
+}
+
+blockquote,
+
+q {
+
+  quotes: none;
+
+}
+
+blockquote:before,
+
+blockquote:after,
+
+q:before,
+
+q:after {
+
+  content: "";
+
+  content: none;
+
+}
+
+table {
+
+  border-collapse: collapse;
+
+  border-spacing: 0;
+
+}
+
+
+
+/* Set every element's box-sizing to border-box */
+
+* {
+
+  box-sizing: border-box;
+
+}
+
+
+
+html,
+
+body {
+
+  height: 100%;
+
+  font-family: "Titillium Web", sans-serif;
+
+}
+
+
+
+h1,
+
+h2,
+
+h3,
+
+h4,
+
+h5 {
+
+  font-family: "Bangers", cursive;
+
+  letter-spacing: 1px;
+
+  margin-bottom: 15px;
+
+}
+
+
+
+/* Your code starts here! */
+.container {
+
+	width: 876px;
+
+	margin: 0px auto;
+
+}
+
+.nav-content {
+
+	display: flex;
+
+	justify-content: space-between;
+
+	align-items: center;
+
+	margin-top: 30px;
+
+}
+
+nav {
+
+	display: flex;
+
+	width: 69%;
+
+	justify-content: space-between;
+
+}
+
+.nav-button {
+
+	text-decoration: none;
+
+	color: grey;
+
+	transition: transform 0.4s;
+
+}
+
+.nav-button:hover {
+
+	transform: scale(1.2);
+
+}
+
+.cta {
+
+	width: 100%;
+
+	display: flex;
+
+	justify-content: space-between;
+
+	margin-top: 65px;
+
+	padding-left: 126px;
+
+	margin-bottom: 79px;
+
+}
+
+.cta-text {
+
+	text-align: center;
+
+	display: flex;
+
+	flex-direction: column;
+
+	justify-content: center;
+
+}
+
+.innovation {
+
+	font-size: 72px;
+
+	margin-bottom: 30px;
+
+}
+
+.get-started a {
+
+	text-decoration: none;
+
+	border: 1px solid black;
+
+	padding: 3px;
+
+	padding-left: 45px;
+
+	padding-right: 45px;
+
+	color: black;
+
+	transition: 0.5s;
+
+	font-size: 17px;
+
+}
+
+.get-started a:hover {
+
+	background: green;
+
+	border:1px solid darkgreen;
+
+	color: black;
+
+}
+
+.top-content {
+
+	border-top: 1px solid black;
+
+	display: flex;
+
+	padding-top: 35px;
+
+}
+
+.middle-img {
+
+	padding-top: 32px;
+
+	padding-bottom: 26px;
+
+}
+
+.bottom-content {
+
+	border-bottom: 1px solid black;
+
+	padding-bottom: 38px;
+
+	display: flex;
+
+}
+
+.bottom-content div {
+
+	width: 31.2%
+
+}
+
+.services, .product {
+
+	margin-right: 20px;
+
+}
+
+.contact {
+
+	margin-top: 33px;
+
+}
+
+.contact p {
+
+	margin-bottom: 12px;
+
+}
+
+address {
+
+	display: flex;
+
+	flex-direction: column;
+
+	align-content: space-between;
+
+}
+
+.copyright {
+
+	text-align: center;
+
+	margin-top: 28px;
+
+	margin-bottom: 40px;
+
+	font-size: 16px;
+
+}
+
+@media (max-width: 800px) {
```

Comment:
Way to keep the css organized, If that helps, do the media max/min with at the bottom